query_optimizer/ExecutionGenerator.cpp

```diff
@@ -830,26 +830,23 @@ void ExecutionGenerator::convertCreateIndex(
     index_attributes.emplace_back(catalog_attribute);
   }
 
-  // Corresponding to each attribute, create a copy of index description.
-  std::vector<IndexSubBlockDescription> index_descriptions;
+  // Create a copy of index description and add all the specified attributes to it.
```

Comment:
Since you're here, refactor `input_relation` in Line 812 to

```
  const CatalogRelation &input_relation = *input_relation_info->relation;
```

and then replace `input_relation->` with `input_relation.`.